src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -283,15 +283,15 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig,
     val lifecycle = new Lifecycle().setRule(List(lifecycleRule).asJava)
 
     // The Leo service account
-    val leoServiceAccountEntityString = s"user-${dataprocConfig.serviceAccount.string}"
+    val leoServiceAccountEntityString = s"user-${dataprocConfig.serviceAccount.value}"
```

Comment:
I [confirmed that this is indeed](https://developers.google.com/resources/api-libraries/documentation/storage/v1/java/latest/com/google/api/services/storage/model/BucketAccessControl.html#setEntity(java.lang.String)) `user-foo@bar.baz`. (I still find this incredibly weird.)